content/community/contributing.md

```diff
@@ -51,14 +54,16 @@ The only way to fix a bug, is to reproduce it first. At the very least, provide
 If you can, write a Unit Test or a Cucumber Feature that demonstrates the bug.
 On occasion, we might ask you to provide a [Minimal, Complete, and Verifiable example](https://stackoverflow.com/help/mcve) (preferably in the form of a Github project).
 
-If you file a Github issue without providing following these quidelines, your ticket may be closed. If that happens, please provide the missing information in the ticket to have it reopened.
+If you file a Github issue without providing following these guidelines, your ticket may be closed. If that happens, please provide the missing information in the ticket to have it reopened.
```

Comment:
Maybe we should also remove "providing"? `...without providing following these guidelines... `

Reply:
Nice catch! Will do